src/localisation-service.ts

```diff
@@ -15,6 +15,34 @@ export class LocalisationService {
     }
   }
 
+  get shortDateFormat() {
+    switch (this.locale) {
+      case 'en-US':
+        return 'MM/DD/YYYY'
+      case 'en-AU':
+      default:
+        return 'DD/MM/YYYY'
+    }
+  }
+  get longDateFormat() {
+    switch (this.locale) {
+      case 'en-US':
+        return 'MMM Do, YYYY'
+      case 'en-AU':
+      default:
+        return 'Do MMM, YYYY'
+    }
+  }
+  get longDateTimeFormat() {
```

Comment:
can these be renamed to have the library they are for.

Reply:
Pretty sure its universal. Works for moment, mui and looks like will work for date-fns too

Reply:
My mistake, its slightly different for date-fns. mui must be using the formatter for whichever lib you use with it